mkdocs/tests/build_tests.py

```diff
@@ -475,6 +477,55 @@ def test_copy_theme_files(self, site_dir, docs_dir):
         self.assertPathNotExists(site_dir, 'content.html')
         self.assertPathNotExists(site_dir, 'main.html')
 
+    def test_build_error_exception(self):
```

Comment:
I understand that this test takes the form of similar tests in `build_tests.py`. However, it is a test related to the plugin API and that is where one would expect to find it. Lets move the test to `plugin_tests.py`.

Reply:
Sure, moved into `plugin_tests.py` :slightly_smiling_face: